helm/ffc-sfd-permissions-api/values.yaml

```diff
@@ -14,12 +14,12 @@ deployment: {}
 # These may be changed. See here: https://github.com/DEFRA/adp-helm-library?tab=readme-ov-file#container-template
 container:
   memCpuTier: L
-  port: 3000
+  port: 3009
```

Comment:
This should be `3008` to match the Dockerfile as that is what the container runs under